ksqldb-serde/src/test/java/io/confluent/ksql/logging/processing/LoggingDeserializerTest.java

```diff
@@ -60,6 +61,7 @@
   @Before
   public void setUp() {
     deserializer = new LoggingDeserializer<>(delegate, processingLogger);
+    deserializer.configure(Collections.emptyMap(), false);
```

Comment:
Probably worth adding a test that called `configure` with `isKey=true`, and ensure logged error has `isKey=true`.

Same for `LoggingSerializerTest`.